src/components/application_manager/src/message_helper/message_helper.cc

```diff
@@ -2418,8 +2419,10 @@ void MessageHelper::SendOnPermissionsChangeNotification(
   content[strings::msg_params] =
       smart_objects::SmartObject(smart_objects::SmartType_Map);
 
-  // content[strings::msg_params][strings::app_id] = connection_key;
-
+  if (encryprion_required.is_initialized()) {
+    content[strings::msg_params][strings::requireEncryption] =
```

Comment:
Please keep smart object keys naming conventions. Should be "require_encryption"

Reply:
@JackLivio, Right, fixed in fixup commits 18ca363907f4b5083684603550f5f06ef6cd484b and 294d2c9928eb4f3b21fce04d6fc27c62b3592be3 which will be squashed after your approval